Mathlib/Topology/Algebra/Module/StrongTopology.lean

```diff
@@ -495,13 +495,20 @@ end BoundedSets
 
 section BilinearMaps
 
-variable {𝕜 𝕜₂ 𝕜₃ : Type*} [NormedField 𝕜] [NormedField 𝕜₂] [NormedField 𝕜₃] {E F G : Type*}
-  [AddCommGroup E] [Module 𝕜 E] [TopologicalSpace E]
+variable {𝕜 𝕜₂ 𝕜₃ : Type*} [Semiring 𝕜] [NormedField 𝕜₂] [NormedField 𝕜₃] {E F G : Type*}
+  [AddCommGroup E] [Module 𝕜 E]
   [AddCommGroup F] [Module 𝕜₂ F] [TopologicalSpace F]
   [AddCommGroup G] [Module 𝕜₃ G]
   [TopologicalSpace G] [IsTopologicalAddGroup G] [ContinuousConstSMul 𝕜₃ G]
   {σ₁₃ : 𝕜 →+* 𝕜₃} {σ₂₃ : 𝕜₂ →+* 𝕜₃}
 
+/-- Send a bilinear map continuous in the second variable to a bilinear map. -/
+@[simps!]
+def _root_.LinearMap.toLinearMap₂ (L : E →ₛₗ[σ₁₃] F →SL[σ₂₃] G) : E →ₛₗ[σ₁₃] F →ₛₗ[σ₂₃] G :=
```

Comment:
I would like to have a reference to CLM in the name. Otherwise it looks like a purely algebraic definition.

Reply:
Would `_root_.LinearMap.CLMtoLinearMap₂` work?

Reply:
Yeah, definitely better imo, but I am terrible at naming things in mathlib.

Reply:
> I am terrible at naming things in mathlib.

I'm glad it's not just me who struggles to think up good names!

Reply:
Nice, good point! Sorry for the noise.